sno/working_copy/gpkg.py

```diff
@@ -73,7 +53,7 @@ def _sno_table(self, name, suffix=""):
         n = f"{self.SNO_TABLE_PREFIX}{name}"
         if suffix:
             n += "_" + suffix
-        return gpkg.ident(n)
+        return n
 
     @contextlib.contextmanager
     def session(self, bulk=0):
```

Comment:
We should be able to ditch this whole context manager and use [SQLAlchemy's transaction handling](https://docs.sqlalchemy.org/en/14/core/connections.html#using-transactions)?

Reply:
Had a quick try at this but gave up for right now - took me a while of experimenting before to get it working so that tests pass.
The code will in the end be very similar to this anyway right - all I am hoping is to switch these few lines of code:
```
  db.execute("BEGIN TRANSACTION")
  yield db
  db.commit()
except Exception:
  db.rollback()
```
goes to something like:
```
with connection.begin():
  yield connection
```

which would be nice, but doesn't change the code very much, and I tried it right now and it made some tests fail - possibly because i need to add the workarounds suggested here:
https://docs.sqlalchemy.org/en/14/dialects/sqlite.html#pysqlite-serializable

For now I'll add a TODO and try and close off this PR.